apps/server/src/modules/learnroom/controller/dto/single-column-board/board-column-board.response.ts

```diff
@@ -29,4 +30,7 @@ export class BoardColumnBoardResponse {
 
 	@ApiProperty()
 	columnBoardId: string;
+
+	@ApiProperty()
+	layout: string;
```

Comment:
should this not be also BoardLayout enum?

Reply:
Right, changed that.